contrib/phrases/phrases.py

```diff
@@ -0,0 +1,155 @@
+# coding: utf8
+from __future__ import unicode_literals
+import sys
+
+import prodigy
+from prodigy.core import recipe_args
+from prodigy.components.db import connect
+from prodigy.components.sorters import Probability
+from prodigy.util import log, prints, split_string, set_hashes
+import requests
+import spacy
+import srsly
+
+
+@prodigy.recipe('phrases.teach',
+    dataset=recipe_args["dataset"],
+    seeds=("One or more comma-separated seed terms", "option", "se", split_string),
+    threshold=("Similarity threshold for sense2vec", "option", "t", float),
+    batch_size=("Similarity threshold for sense2vec", "option", "bs", int),
+    resume=("Resume from existing phrases dataset", "flag", "R", bool)
+)
+def phrases_teach(dataset, seeds, threshold=0.85, batch_size=5, resume=False):
+    """
+    Bootstrap a terminology list with word vectors and seeds terms. Prodigy
+    will suggest similar terms based on the word vectors, and update the
+    target vector accordingly.
+    """
+
+    DB = connect()
+    seed_tasks = [set_hashes({"text": s, "answer": "accept"}) for s in seeds]
+    DB.add_examples(seed_tasks, datasets=[dataset])
+
+    accept_phrases = seeds
+    reject_phrases = []
+
+    seen = set(accept_phrases)
+    sensed = set()
+
+    if resume:
+        prev = DB.get_dataset(dataset)
+        prev_accept = [eg["text"] for eg in prev if eg["answer"] == "accept"]
+        prev_reject = [eg["text"] for eg in prev if eg["answer"] == "reject"]
+        accept_phrases += prev_accept
+        reject_phrases += prev_reject
+
+        seen.update(set(accept_phrases))
+        seen.update(set(reject_phrases))
+
+    def sense2vec(phrase, threshold):
+        """Call sense2vec API to get similar "senses" (phrases)"""
+        res = requests.post('https://api.explosion.ai/sense2vec/find', {
```

Comment:
Maybe we could move this URL to the top as a variable `API_URL` or something? And then add a comment mentioning the repo and how to self-host the service?

It's fine if people use our API for testing, but I just don't want to encourage workflows that depend on services hosted by us that may change in the future (and possibly break people's code).

Reply:
Updated this. Included a blurb in the README.md